src/yashCJun20/Assignment25/SumOfAllNumbersInString.java

```diff
@@ -0,0 +1,23 @@
+package yashCJun20.Assignment25;
+
+public class SumOfAllNumbersInString {
+	
+	void toCalculateSumOfAllNumbersInString (String name) {
+		
+			int sum=0;
+			for(int i=0;i<name.length();i++) {
+				if(Character.isDigit((name.charAt(i)))) {
+					String charValue = String.valueOf(name.charAt(i));
+					int digit = Integer.parseInt(charValue);
```

Comment:
Instead of line number 10 and 11. you can write below line.

You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));